lib/appmap/handler/rails/sql_handler.rb

```diff
@@ -43,6 +43,20 @@ class << self
             def examine(payload, sql:)
               return unless (examiner = build_examiner)
 
+              if AppMap.explain_queries? && examiner.in_transaction? && examiner.database_type == :postgres
+                unless sql =~ /\A(SAVEPOINT|RELEASE|ROLLBACK|BEGIN|INSERT|COMMIT)/i
```

Comment:
That selection is somewhat haphazard. Why exclude `INSERT` but include `CREATE`, for example? I think it would make more sense to simply have a whitelist of the four query commands, ie. `SELECT`, `INSERT`, `DELETE` and `UPDATE`, plus a `WITH` for CTEs (which are only valid at the start in queries).

Reply:
OK, sounds good.